src/modules/keyboard/Config.cpp

```diff
@@ -190,25 +191,40 @@ Config::Config( QObject* parent )
                  emit prettyStatusChanged();
              } );
 
-    connect( m_keyboardVariantsModel, &KeyboardVariantsModel::currentIndexChanged, this, &Config::xkbChanged );
+    connect( m_keyboardVariantsModel,
+             &KeyboardVariantsModel::currentIndexChanged,
+             [ & ]( int index )
+             {
+                 m_selectedVariant = m_keyboardVariantsModel->key( index );
+                 Config::xkbChanged();
```

Comment:
Since you are now using a lambda with implicit capture of `this`, you can just call member functions, like `xkbChanged()`. No need for the weird syntax here.

The reason it said `&Config::xkbChanged` in the first place is that that was a member-function-pointer, which has specific syntax.